docs/topics/modules/e2e/create_springboot.adoc

```diff
@@ -1,15 +1,23 @@
 [#create_springboot]
 = Creating a Spring Boot Quickstart Project
 
+You will not learn how to create a second quickstart project in your Space. The Hello World project was a Vert.X application and the second quickstart is a Spring Boot application.
+
 . In your {osio} home page, double-click your new space.
 . In your space's dashboard, click btn:[Add to space].
-. Select `Create a new Quickstart project`.
+. Click btn:[Create a new Quickstart project].
++
+image::get_started_menu.png[Getting Started Menu]
++
 . In the `Application Wizard`:
-.. Click a project type from the displayed icons. For this example, click btn:[Spring Boot - HTTP].
+.. In the `Name` field, type *spring* and press kbd:[Enter] to filter the options for Spring Boot quickstart applications.
+.. From the displayed results, click btn:[Spring Boot - HTTP].
```

Comment:
I don't think this thing qualifies as a 'button'. Maybe call it 'item' or just 'option'?